src/components/Navigation/PollNavigationItems.vue

```diff
@@ -9,12 +9,12 @@ import { t } from '@nextcloud/l10n'
 import NcActionButton from '@nextcloud/vue/components/NcActionButton'
 import NcAppNavigationItem from '@nextcloud/vue/components/NcAppNavigationItem'
 
-import DeletePollIcon from 'vue-material-design-icons/Delete.vue'
+import DeletePollIcon from 'vue-material-design-icons/DeleteOutline.vue'
```

Comment:
same, would be `TrashCanOutline` since we decided to go for Material Symbols, and from the Material Design Icons that would be the matching icon